docs/development/extensions-core/kinesis-ingestion.md

```diff
@@ -719,9 +406,16 @@ If resharding occurs when the supervisor is suspended and `useEarliestSequence`
 
 ## Kinesis known issues
 
-Before you deploy the Kinesis extension to production, consider the following known issues:
+Before you deploy the `druid-kinesis-indexing-service` extension to production, consider the following known issues:
```

Comment:
Might be worth checking with druid devs to see if any of these have been addressed.

Reply:
Checked and updated.